lib/ae_page_objects/core/dsl.rb

```diff
@@ -20,7 +20,9 @@ def element(name, options = {}, &block)
 
       self.element_attributes[name.to_sym] = klass
 
-      define_method name do |&block|
+      define_method name do |&block_for_define_method|
```

Comment:
@dtognazzini 
ok, I am fine with just remove the block handling without throwing exception if a block is passed, just feel that it's so easy to pass block which contains the assertion test by mistake(the test just pass since it never run :cry: ) maybe that case should be handled by developer not by aepos.